internal/json/json.go

```diff
@@ -20,10 +25,92 @@ type Converter struct {
 	Filename             string
 	Package, MessageType string
 	Indent               bool
+	EndOfMessageMarker   string
 }
 
 // Convert converts proto message to json.
 func (c Converter) Convert(r io.Reader) ([]byte, error) {
+	md, err := c.createProtoMessageDescriptor()
+	if err != nil {
+		return nil, err
+	}
+
+	b, err := ioutil.ReadAll(r)
+	if err != nil {
+		return nil, err
+	}
+
+	parsed, err := c.unmarshalProtoBytesToJson(md, b)
+	if err != nil {
+		return nil, err
+	}
+	return parsed, nil
+}
+
+// ConvertStream converts multiple proto messages to json.
+// It returns a result channel and error channel which both can return multiple messages (a result or error for each message)
+// Because proto messages often contain newlines, we can't rely on new lines for knowing when one message ends and the
+// next begins, so instead it looks for a line containing only a specified marker (defaults to DefaultEndOfMessageMarker).
+// Although unlikely, it is possible that the EndOfMessageMarker can be part of the proto binary message, in which case the
+// parsing of that message will fail. If this happens, use a more complex EndOfMessageMarker.
+func (c Converter) ConvertStream(r io.Reader) (resultCh chan []byte, errorCh chan error) {
```

Comment:
I also suggest to return `receiver channels`:
```suggestion
func (c Converter) ConvertStream(r io.Reader) (resultCh <-chan []byte, errorCh <-chan error) {
```